pkg/project/registry/project/proxy/proxy.go

```diff
@@ -128,6 +128,29 @@ func (s *REST) Create(ctx kapi.Context, obj runtime.Object) (runtime.Object, err
 	return convertNamespace(namespace), nil
 }
 
+func (s *REST) Update(ctx kapi.Context, obj runtime.Object) (runtime.Object, bool, error) {
```

Comment:
we need a test to make sure a project admin can't update their project/namespace until we get per-field restrictions in place (to prevent them changing node selectors or scc uid allocations)